src/shared/App.js

```diff
@@ -2,16 +2,30 @@ import React from "react";
 import {Route, Switch} from 'react-router-dom'
 import {Home, About, Class, Login, Register} from "../routes";
 import {Header, Footer} from "../components/index.js";
+import "./../styles/styles.css"
 
 class App extends React.Component {
+    state = {
+        token:null
+    }
+    componentDidMount(){
+        const token = localStorage.getItem("t2t-token")
+        if(token){
+            this.setState({token})
+        }
+    }
     // 라우팅은 여기서 진행
     render() {
         return (
             <div className="App">
                 <Route path={'/'} component={Header} />
                 <Route exact path='/' component={Home}/>
-                <Route path='/user/login' component={Login}/>
-                <Route path='/user/register' component={Register}/>
+                {
+                 !this.state.token && <Route path='/user/login' component={Login}/>
+                }
+                {
+                !this.state.token && <Route path='/user/register' component={Register}/>
+                }
```

Comment:
That's right.
# Changes 
- I will add a new Home Page which will contain text as you are authenticated
- Whenever user login, they will be redirected to Home Page
- If they are already login (means already have a valid token), they will be automatically redirected to Home Page
- In case a user who is already login try to visit signin/register page he will be redirected to Home Page
- Any unauthorized user cannot visit Home Page(if tries will automatically redirected to login page).
- I will implement redux

@jinwoo1225 can you please confirm should i make these changes